src/services/index/compare-questions-index/compare-questions-index-service.ts

```
@@ -0,0 +1,176 @@
+import { getQuestionHash } from "../../../core/index/getHash";
+import QuestionEntity from "../../../domain/index/entities/Question";
+
+import { Question } from "../../../infra/eos/contracts/peerania.dev/tables/Question";
+import { getTableRows } from "../../../infra/eos/eos-utils";
+
+import { getConnection } from "../../../infra/mysql/methods";
+
+import {
+  handleCreatedQuestion,
+  handleDeletedQuestion,
+  handleUpdatedQuestion,
+} from "./question-handling";
+
+import { IItems } from "./interfaces";
+
+const loadingNumber = 1000;
+
+const connection = getConnection();
+
+const repository = connection.then((x) => x.getRepository(QuestionEntity)).catch(() => {
+  throw new Error("Error in repository getting (orm, mysql)");
```

Hiding original error. It will be difficult to troubleshoot if an issue occurs here.
Original error needs to be logged. What is the reason for throwing a generic error?